central/ui/mux.go

```diff
@@ -1,14 +1,20 @@
 package ui
 
 import (
+	"fmt"
 	"net/http"
+
+	"github.com/stackrox/rox/pkg/version"
 )
 
 // Mux returns a HTTP Handler that knows how to serve the UI assets,
 // including Javascript, HTML, and other items.
 func Mux() http.Handler {
 	mux := http.NewServeMux()
-	mux.Handle("/docs/product/", http.StripPrefix("/docs/product/", http.FileServer(http.Dir("/stackrox/static-data/product-docs"))))
+	mux.Handle("/docs/product/", http.StripPrefix("/docs/product/",
+		http.RedirectHandler(fmt.Sprintf("https://docs.openshift.com/acs/%s/welcome/index.html",
+			version.GetMajorMinor(version.GetMainVersion())),
```

Comment:
Is it worthwhile to write a short comment about the possible future scenario which you brought up, if I remember correctly: if **dev build** is looking forward to next release instead of looking backward to version number of previous release? In that hypothetical numbering convention, dev build 3.74.x-whatever would need to link to 3.73 last published version of docs instead of not-yet-existing 3.74 version, correct?

Reply:
Correct. That is a good point.